[BREAKING] do not serialize fields abi, devdoc, userdoc and storage-layout (#10409)
ethereum/solidity: Changelog.md

 * Type System: Explicit conversions from literals to integer type is as strict as implicit conversions.
 * Type System: Explicit conversions from literals to enums are only allowed if the value fits in the enum.
 * Type System: Declarations with the name ``this``, ``super`` and ``_`` are disallowed, with the exception of public functions and events.

>  * Command Line Interface: Content of JSON-Fields `abi`, `devdoc`, `userdoc` and `storage-layout` are sub-objects now.

Reviewer: And a similar change here.

```suggestion
 * Command Line Interface: JSON fields `abi`, `devdoc`, `userdoc` and `storage-layout` are now sub-objects rather than strings.
```
